profiler/profiler.go

```diff
@@ -17,6 +17,13 @@ import (
 	"github.com/longhorn/go-common-libs/utils"
 )
 
+// Op is the human readable operation for the profiler
+const (
+	OpSHOW    = "SHOW"
+	OpENABLE  = "ENABLE"
+	OpDISABLE = "DISABLE"
+)
```

Comment:
Since these constants are only used by the unit test. Do we still want to have them? If so, what do you think about moving it over to profiler_test.go?

Reply:
First, I would like to expose these three Ops to the caller.
But it seems the caller should handle it themself. Moved these variables to the test file.